benchmarks/vectorized_jittered_resize.py

```diff
@@ -87,15 +87,15 @@ class OldJitteredResize(BaseImageAugmentationLayer):
             This factor is used to scale the input image.
             To replicate the results of the MaskRCNN paper pass `(0.8, 1.25)`.
         crop_size: (Optional) the size of the image to crop from the scaled
-            image, defaults to `target_size` when not provided.
+            image. Defaults to `target_size`.
         bounding_box_format: The format of bounding boxes of input boxes.
             Refer to
-            https://github.com/keras-team/keras-cv/blob/master/keras_cv/bounding_box/converters.py
```

Comment:
why drop the github.com prefix?

Will GitHub automagically render this link for us in the UI? That would be cool

Reply:
@ianstenbit Oh actually I think it was flake8 picking up a long line